server/jetstream_cluster_test.go

```diff
@@ -1168,15 +1170,16 @@ func TestJetStreamClusterConsumerInfoList(t *testing.T) {
 	}
 
 	// Now get the consumer list info.
-	cl := js.NewConsumerLister("TEST")
-	if !cl.Next() {
-		t.Fatalf("Unexpected error: %v", cl.Err())
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	var infos []*nats.ConsumerInfo
+	for info := range js.ConsumersInfo(ctx, "TEST") {
```

Comment:
```suggestion
	for info := range js.ConsumersInfo(nil, "TEST") {
```

Reply:
to simplify the testing code we should use the `nil` default and rely on the iterator/page stop